.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: go
 go:
-  - 1.7.3
+  - 1.9.x
```

Comment:
Hi @dvrkps Thanks for your contribution. At present, our plugin also support to go version 1.8.x and I think it is too early to update it to version 1.9.x. Thank a lot.

Regard
He Guimin